src/components/NavBar/NavBar.tsx

```diff
@@ -69,13 +82,15 @@ const NavBar = () => {
             src={`/chirp-logo-transparent.png`}
             sx={styles.logo}
           />
-          <List component="nav">
+          <List sx={styles.list} component="nav">
             {navItems.map((navItem, index) => (
               <NavItem
                 key={index}
                 icon={navItem.icon}
+                altIcon={navItem.altIcon}
                 label={navItem.label}
                 route={navItem.route}
+                active={location.pathname === navItem.route}
```

Comment:
I kind of think `selected` is a more descriptive name, what do you think?

Reply:
Yes, it is. I have updated the change.